.github/actions/ortdocker/action.yml

```diff
@@ -65,9 +65,6 @@ runs:
         result=$(python ./.github/actions/ortdocker/check_image.py)
         echo "result=$result" >> $GITHUB_OUTPUT
 
-    - name: Set up QEMU
```

Comment:
Please explain in the commit message *why* this is reverted, i.e. what problem it caused.

Reply:
Done